monai/handlers/checkpoint_saver.py

```diff
@@ -116,7 +116,8 @@ class _DiskSaver(DiskSaver):
             """
 
             def __init__(self, dirname: str, filename: Optional[str] = None):
-                super().__init__(dirname=dirname, require_empty=False)
+                # set `atomic=False` as `atomic=True` may save files without read permission for non-root users
```

Comment:
More exactly, it gives read/write permission to the user who saved the file, without group/others read permission, i.e. `-rw-------`, not related to root or not-root user...

Reply:
Hi @vfdev-5 , thanks for your review. I updated the comments.